versions.tf

```diff
@@ -1,14 +1,14 @@
-terraform {
-  required_version = ">= 0.13.0"
-
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = ">= 2.0"
-    }
-    random = {
-      source  = "hashicorp/random"
-      version = ">= 2.1"
-    }
-  }
+terraform {
+  required_version = "~> 1.5.0, <=1.5.5"

```

Comment:
I think we're fine to use versions > 1.5.5 now, so `~> 1.0` would work

Reply:
~> 1.0 was breaking, however ~>1.5.0 works